advanced-cache.php

```diff
@@ -432,7 +512,7 @@ function add_debug_html_to_output( $debug_html ) {
 
 
 // Did we find a batcached page that hasn't expired?
-if ( isset($batcache->cache['time']) && ! $batcache->genlock && time() < $batcache->cache['time'] + $batcache->cache['max_age'] ) {
+if ( isset($batcache->cache['time']) && ! $batcache->genlock && ( time() < $batcache->cache['time'] + $batcache->cache['max_age'] || ( $batcache->use_stale && time() < $batcache->cache['time'] + $batcache->cache['max_age'] + $batcache->stale_max_age ) || ( $batcache->use_stale && $batcache->stale_max_age === 0 ) ) ) {
```

Comment:
Parenthesis inconsistency: the expired check is not wrapped in parentheses, but the use_stale checks both are.

(The inconsistency is just a style issue: it doesn't cause any functional issue.)